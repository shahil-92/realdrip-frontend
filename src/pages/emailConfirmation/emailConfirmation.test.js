import React from 'react';
import EmailConfirmation from '.';

describe('<EmailConfirmation />', () => {
  describe('Rendering', () => {
    it('should render correctly', () => {
      const wrapper = shallow(<EmailConfirmation />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});

