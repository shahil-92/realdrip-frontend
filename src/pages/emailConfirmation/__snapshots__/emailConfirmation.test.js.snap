// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmailConfirmation /> Rendering should render correctly 1`] = `
<div>
  <h1>
    Please confirm your email account.
  </h1>
</div>
`;
